### ISSUE TITLE
Diarization process in Whisperx does not utilize GPU

### ISSUE THREAD

**Issue by SerebryanskiySergei** [m-bain/whisperX#542]
I have updated the package to the latest version with the merged 3.0.1 version of pyannite audio. 
However, I am still experiencing slow diarization processing times.

After checking the Task Manager, I noticed that only 1-2% of GPU usage is observed (I am using a 4080 GPU). 
In the same time transcription process takes almost 98% so I'm sure that the setup is correct.

Do you have any ideas on how to resolve this issue?

**Comment by sam1am** [comment]
See here: https://github.com/m-bain/whisperX/issues/499

**Comment by SerebryanskiySergei** [comment]
> See here: #499

yeah, I've checked that before making an issue, thank you for pointing this out.

Maybe I didn't get the idea, but the author is talking there about 3.0.0 version that doesn't work on the GPU. 
However with the merged fix from the prev week, pyannnite.audio was updated to 3.0.1 and this version whould be OK with the GPU usage.

But the problem is still exists.

**Comment by iamianM** [comment]
Hi, thanks for bringing this up. I'm having the same issue. Was hoping the update would resolve, but still stuck using the CPU.

**Comment by manjunath7472** [comment]
Assuming cuda and cudnn installed in PC.

Before installing whisperx, install pytorch for your cuda version installed in your pc.
For cuda 11.8
pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118

Then install whisperX

post install, confirm cuda by running below code.
```
import torch
print(torch.cuda.is_available())
```

**Comment by SerebryanskiySergei** [comment]
> Assuming cuda and cudnn installed in PC.
> 
> Before installing whisperx, install pytorch for your cuda version installed in your pc. For cuda 11.8 pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118
> 
> Then install whisperX
> 
> post install, confirm cuda by running below code.
> 
> ```
> import torch
> print(torch.cuda.is_available())
> ```

I thought about it too, as an example I've built with cuda official example applications (deviceQuery), I was able to build and run it.

**Comment by Barabazs** [comment]
Closing as duplicate of #274 which tracks diarization GPU utilization issues.
